src/v2/models/AppState.ts

```diff
@@ -222,7 +222,7 @@ export default class AppState extends Model {
     const currentFormName = this.get('currentFormName');
     const currentViewState = transformedResponse.remediations.filter(r => r.name === currentFormName)[0];
     // Get new refresh interval for either: remediations, authenticator, or authenticator enrollment
-    return currentViewState.refresh ||
+    return currentViewState?.refresh ||
```

Comment:
TestCafe 3 produces
```
      JavaScript error details:
      TypeError: Cannot read properties of undefined (reading 'refresh')
          at AppState.getRefreshInterval (http://localhost:3000/js/okta-sign-in.js:63861:29)
          at AppState.shouldReRenderView (http://localhost:3000/js/okta-sign-in.js:63850:47)
          at AppState.setIonResponse (http://localhost:3000/js/okta-sign-in.js:63921:29)
          at updateAppState (http://localhost:3000/js/okta-sign-in.js:63164:18)
          at child.handleIdxResponse (http://localhost:3000/js/okta-sign-in.js:63579:38)
          at child.invokeAction (http://localhost:3000/js/okta-sign-in.js:63395:16)
          at async child.handleInvokeAction (http://localhost:3000/js/okta-sign-in.js:63373:5)
```
but test can pass after using `?.`

Reply:
Do you have any idea why `currentViewState` is now `undefined` when running TestCafe 3?

Reply:
Add correct mock for 'http://localhost:3000/idp/idx/authenticators/okta-verify/launch'